superset-frontend/src/setup/setupApp.ts

```diff
@@ -84,4 +85,7 @@ export default function setupApp() {
   // @ts-ignore
   window.jQuery = $;
   require('bootstrap');
+
+  // setup appwide custom error messages
+  setupErrorMessages();
```

Comment:
not sure if you want to nest the call here or in `App.jsx` like `setupApp` and `setupPlugins`. The items in `setupApp` here seems likely to be removed at some point.

Reply:
I put it here because it seems like there are a bunch of `App.jsx`s that exist today, and we'd want all of them to have the error message registry attached to them. I assume in the future we'll end up with only one and this file will be removed, but i think it's fine to keep this here for now